monkey/tests/unit_tests/agent_plugins/exploiters/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.fixture
+def mock_agent_otp_environment_variable():
+    return "MOCK_AGENT_OTP_ENV_VAR"
```

Comment:
This isn't a mock. See https://martinfowler.com/bliki/TestDouble.html

This is just an environment variable.

```suggestion
def agent_otp_environment_variable():
    return "TEST_AGENT_OTP_ENV_VAR"
```